Resources/Prototypes/Entities/Structures/Walls/asteroid.yml

```
@@ -897,6 +919,28 @@
+  id: WallRockTungsten
+  parent: WallRock
+  description: An ore vein rich with wolframite.
+  suffix: Copper
+  components:
+    - type: OreVein
+      oreChance: 1.0
+      currentOre: OreTungsten
+    - type: Sprite
+      layers:
+        - state: rock
+        - map: [ "enum.EdgeLayer.South" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)